SUMMARY.md

```diff
@@ -28,10 +28,13 @@
     * [Github](user-guides/asset-hosting/github.md)
   * [Join a Data Challenge](user-guides/join-a-data-challenge.md)
   * [Sponsor a Data Challenge](user-guides/sponsor-a-data-challenge.md)
-  * [Get Started Data Farming](user-guides/get-started-df.md)
-  * [Estimate your APYs](user-guides/how-to-df-estimate-apy.md)
-  * [Harvest More Yield Data Farming](user-guides/how-to-data-farm.md)
-  * [Claim Rewards Data Farming](user-guides/claim-ocean-rewards.md)
+  * [Data Farming](user-guides/data-farming/README.md)
```

Comment:
Suggestion: The estimate APYs page should be in the rewards section and just linked in the Earn Passive rewards file.

Reply:
Ticket for tracking feedback: https://github.com/oceanprotocol/docs/issues/1233
Let's improve after shipping